src/tree/v2/application/ApplicationResourceTreeDataProvider.ts

```diff
@@ -41,6 +43,19 @@ export class ApplicationResourceTreeDataProvider extends ResourceTreeDataProvide
                 this.statusSubscription?.dispose();
             });
 
+        registerEvent(
+            'treeView.onDidChangeConfiguration',
+            vscode.workspace.onDidChangeConfiguration,
+            async (context: IActionContext, e: vscode.ConfigurationChangeEvent) => {
+                context.errorHandling.suppressDisplay = true;
```

Comment:
Any reason not to just `suppressAll` here? Do errors thrown by listeners to `this.onDidChangeTreeData` end up here?